src/containers/TriggerTemplates/TriggerTemplates.scss

```diff
@@ -0,0 +1,16 @@
+/*
+Copyright 2019 The Tekton Authors
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+div h1 {
```

Comment:
Can we scope this to just this page? It's a very broad selector that could mess with other page layouts, especially for 3rd parties if this ended up getting loading outside the dashboard.